### PR TITLE
Readme changes.

### PR DESCRIPTION
**Pull request by Livila**
Updated the readme file and test files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,55 @@
...
 
 ## Testing ##
 
-### Manually ###
+To run the test files you can run this command with Gradle:
 
-To run the tests run this command:
-
-```bash
+```sh
```
Comment (Livila) [inline README.md:16]: - [] Fungerar `sh` på Windows på samma sätt som i UNIX?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
 ### Features ###
 
-* Query an LLM on complicated situations where 2 functions could be similar
-* Configuration
-* Testing
+* Query an LLM (large language model) on complicated situations where 2 or more functions could be similar.
+* Configuration - it's easy to change the configurations.
+* Comparison - matching through names of lines and functions. 
+* Tested on both UNIX and Windows.
```
Comment (Livila) [inline README.md:11]: Should only UNIX be mentioned or Mac OS and Linux instead?

Comment (Livila) [reply README.md:11]: I just reverted to `bash` instead.

File: README.md [hunk 3]
```diff
@@ -17,32 +18,39 @@ To run the test files you can run this command with Gradle:
...
+### Download the language model ###
+
+These are only suggestions, you may use any source and model as you want.
 
-### Download a language model ###
 * Either download the bin-file directly from [Hugging Face](https://huggingface.co/) which we used for testing:
 [ggml-model-gpt4all-falcon-q4_0.bin](https://huggingface.co/nomic-ai/gpt4all-falcon-ggml/resolve/main/ggml-model-gpt4all-falcon-q4_0.bin?download=true) (4.06 GB)
 
-* Or choose downloads from the website for gpt4all-falcon-ggml:
+* Or choose downloads from their website for gpt4all-falcon-ggml:
```
Comment (Livila) [inline README.md:32]: Should be reverted.

**Review by FeKozma — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (FeKozma) [reply README.md]: Go for it

**Pull request merged**